pyaedt/edb_core/components.py

```diff
@@ -933,7 +934,7 @@ def create_port_on_component(
         return True
 
     @pyaedt_function_handler()
-    def _create_terminal(self, pin):
+    def _create_terminal(self, pin, term_name=None):
```

Comment:
The new parameter must be added to the "Parameters" section, and this section should follow the standard format.

Reply:
even for internal command ?

Reply:
> even for internal command ?

@svandenb-dev The goal is to have all functions, even internal ones, properly documented.